src/main/java/spireMapOverhaul/zones/gremlincamp/monsters/GremlinCook.java

```diff
@@ -0,0 +1,146 @@
+        super.setUpMisc();
+        this.type = EnemyType.NORMAL;
+    }
+
+    @Override
+    public void takeTurn() { //TODO: Add sounds
+        switch (nextMove) {
+            case 0:
+                useShakeAnimation(Settings.ACTION_DUR_MED);
+                Wiz.atb(new AllEnemyApplyPowerAction(this, strAmt, (m) -> new StrengthPower(m, strAmt)));
+                break;
+            case 1:
+                useShakeAnimation(Settings.ACTION_DUR_FAST);
+                Wiz.atb(new AbstractGameAction() {
+                    @Override
+                    public void update() {
+                        isDone = true;
+                        Wiz.forAllMonstersLiving(m -> {
+                            if(!GremlinCook.ID.equals(m.id)) {
+                                m.increaseMaxHp(healAmt, true);
+                                AbstractDungeon.effectList.add(new HealEffect(m.hb.cX, m.hb.cY, healAmt));
+                            }
+                        });
+                    }
+                });
+                break;
+            case 2:
+                useFastAttackAnimation();
+                Wiz.atb(new ApplyPowerAction(AbstractDungeon.player, this, new PlayerPoisonPower(AbstractDungeon.player, poisonAmt), poisonAmt, AbstractGameAction.AttackEffect.POISON));
+                break;
+            case 3:
+                DamageInfo info = new DamageInfo(this, moves.get(nextMove).baseDamage, DamageInfo.DamageType.NORMAL);
+                if (info.base > -1) {
+                    info.applyPowers(this, AbstractDungeon.player);
+                }
+                useSlowAttackAnimation();
+                addToBot(new DamageAction(AbstractDungeon.player, info, AbstractGameAction.AttackEffect.BLUNT_HEAVY));
+                Wiz.atb(new MakeTempCardInDiscardAction(new Dazed(), calcAscensionSpecial(2)));
+        }
+
+        addToBot(new RollMoveAction(this));
+    }
+
+    @Override
+    protected void getMove(int i) {
+        int enemies = Wiz.getEnemies().size();
+
+        //First turn always buffs
+        if(firstMove) {
+            firstMove = false;
+            setMoveShortcut(ALL_BUFF, MOVES[ALL_BUFF]);
+            return;
+        }
+
+        if(!secondMove) {
+            secondMove = true;
+            if(enemies > 1) {
+                setMoveShortcut(ALL_BULK, MOVES[ALL_BULK]);
+                return;
+            }
+        }
+
+        ArrayList<Byte> possibilities = new ArrayList<>();
+        // If not alone, can buff everyone if it wasn't the last move
+        if(enemies > 1 && !lastMove(ALL_BUFF)) {
+            possibilities.add(ALL_BUFF);
+        } else {
+            //If alone and late in combat, can buff itself again (helps player)
+            if(GameActionManager.turn > 5 && !lastMove(ALL_BUFF)) {
+                possibilities.add(ALL_BUFF);
+            }
+        }
+
+        // Turn 3+ can start poisoning player
+        if(GameActionManager.turn > 2 && !lastMove(POISON)) {
```

Comment:
Given the firstMove and secondMove checks above, it's always turn 3+ if you get to this logic.

Doesn't really matter, but in making sure I understood the logic of these moves I saw the simplification, so I figured I'd comment.